spec/support/federal_email_domains_helper.rb

```diff
@@ -0,0 +1,6 @@
+module FederalEmailDomainHelper
+  def default_federal_domains
```

Comment:
WDYT of naming this to indicate it has side effects?
```suggestion
  def set_up_default_federal_domains!
```